### ISSUE TITLE
diam implement question

### ISSUE THREAD

**Issue by ronghuaiyang** [seasonSH/DocFace#17]
hi, seasonSH:
    thanks for your work.
    in your implementaion, the config of diam is:
   # Loss functions and their parameters.
losses = {
    'diam': {'scale': 'auto', 'm': 5.0, 'alpha':1.0}
}
    but the in the am-softmax paper, the m is about 0.35, so, my question is why in your implemention the m is set to 5, it's much larger than the origin am-softmax paper setting ?
    Looking forward for your replay !

**Comment by yang0817manman** [comment]
origin am-softman: s*(costheta - m)
docface am-softmax: s*costheta - m
so the parameter of "m" is different. @ronghuaiyang
